docs/ide-usage.md

```diff
@@ -190,3 +197,6 @@ If you do not wish to use the automated setup, you can follow the [manual setup]
    ```
 
 1. To set up some static analysis tools, refer to [this document](static-analysis.md).
+
+1. To move on to the development phase, refer to [this document](development.md).
+
```

Comment:
Remove this blank line. The original file ends with only a single newline character.